### ISSUE TITLE
Is it possible to construct a prior model from mouse data sources? 

### ISSUE THREAD

**Issue by mariusmessemaker** [saeyslab/nichenetr#21]
Thank you for doing the important work of combining so many human data sources into one accurate ligand-target model! Really impressive! 

In https://github.com/saeyslab/nichenetr/blob/master/vignettes/symbol_conversion.md you describe that to work with mouse data human gene symbols need to be converted to mouse gene symbols. As the prior model of NicheNet was based mostly on human data sources, does that not decrease accuracy of Nichenet predictions? I noticed that Nichenet was used in this way (i.e. converting human to mouse gene symbols) in the Kupfer cell interaction paper that you mention in the Readme: https://www.cell.com/immunity/fulltext/S1074-7613(19)30368-1#secsectitle0075? I can also appreciate that Nichenets predictions were validated experimentally in this paper. However, I am also wondering whether NicheNets prediction accuracy would increase if a new ligand-target prior model is constructed from mouse data sources only? How difficult would it be to generate such a model, and do you think it will substantially increase Nichenet's prediction accuracy?

**Comment by browaeysrobin** [comment]
Hi @mariusmessemaker 

You raise a valid issue here. First, know that we also included some mouse data sources into the general model (at least for the one-to-one orthologs), but these were strongly in the minority. Because this model is indeed tailored more for human, you can expect that it would perform a bit better for human than mouse, but I don't think the difference is very large. In our paper, we evaluated the ligand-target predictions based on >100 ligand treatment datasets of both human and mouse origin, and there is no indication on these datasets that human would work better than mouse.

Of course, you can also explore the possibility of constructing a mouse-specific model with only mouse data sources. This is some work, but is not so difficult. In the vignette https://github.com/saeyslab/nichenetr/blob/master/vignettes/model_construction.md, we show how to build a model yourself. The main thing to consider though, is that you should have enough primary data sources of mouse to do this. 

So, and this is valid for applying NicheNet to all other species other than human,  you should make the tradeoff between the homology between species, and the number and quality of data sources that are available to build a new species-specific model. For example, for mouse, you could use the human model, but for Drosophila making a new model from own data sources would be more appropriate.

In conclusion: it is possible to build a prior model only from mouse data sources, but I doubt it would work better than using the human model.

**Comment by mariusmessemaker** [comment]
Thank you for your extensive answer @browaeysrobin! I will try the human model first.
